app/controllers/users_controller.rb

```diff
@@ -5,22 +5,24 @@ def index; end
 
   def show
     @user = User.find_by(id: params[:id])
-    render json: @user
   end
 
+  def new; end
+
   def create
     @user = User.create(user_params)
     if @user.save
       flash[:success] = t("create.success_message", name: t("create.label_name"))
+      redirect_to get_user_path(@user)
```

Comment:
```suggestion
      redirect_to @user
```
ri dc ko, mà có action tên là get_user à?

sao em ko dùng action do restful tạo ra nhỉ